src/python/tensorflow_cloud/__init__.py

```diff
@@ -19,3 +19,8 @@
 from .core.machine_config import MachineConfig
 from .core.run import run
 from .core.run import remote
+
+# Provides direct access to Tuner and CloudOracle classes,
```

Comment:
This comment should be directed towards users. Something along the lines of "CloudOracle and CloudTuner APIs for hyperparamter tuning."

Reply:
Done